grammars/javascript.cson

```diff
@@ -126,7 +126,11 @@
   }
   {
     'comment': 'ES6 export: `export default (variable|class|function, etc.)`'
-    'match': '(?x) \\b(export)\\b \\s* \\b(default)\\b (?:\\s*) \\b((?!\\bfunction\\b|\\bclass\\b|\\blet\\b|\\bvar\\b|\\bconst\\b)[a-zA-Z_$][a-zA-Z_$0-9]*)?\\b'
+    'match': '''(?x)
+      \\b(export)\\b   \\s*
```

Comment:
Nope. The space only serves to distinguish one column from another:

<img src="https://cloud.githubusercontent.com/assets/2346707/17825561/99260c2a-66ac-11e6-96b0-e0b4a2b09beb.png" width="203" alt="Figure 1" />

Of course, I can always remove it if you prefer. I just found it pronounced the pertinent parts better for new readers.